### ISSUE TITLE
SSN/SOSA-Core/ Host vs Platform vs Device vs Observer

### ISSUE THREAD

**Issue by kjano** [w3c/sdw#336]
This issue addresses our ongoing discussion about hosts, platforms, devices, and observers. At the core of the issue is the need to come up with a **label** for something that we seem to agree on _conceptionally_, namely 'the thing that carries/hosts sensors, actuators, and so forth'. Examples of this thing include humans and their senses as well as smartphones and their sensors, e.g., an accelerometer.

It is important to recall that we agreed on having a reduced, i.e., surface, axiomatization for SOSA-Core. Put differently, while we can pick any label and define what it should mean, e.g., use the label 'cat' to refer to dogs and 'dog' to refer to cats, the used knowledge representation language fragment and the lack of a deeper axiomatization will prevent us from stating this efficiently. Consequently, the _default_ meaning of the used labels will matter more than we may be comfortable with or willing to accept. We also need to be careful to avoid over generalizing for the sake of being inclusive as we may end up with stating almost nothing and thus hamper semantic interoperability.

Hence, for instance, the term _Device_ should be avoided as it does not include humans . This is an important point as we explicitly decided to include human senses and simulations when we defined _Sensor_.

We also argued against the _Observer_ as it may be perceived as excluding actuators, thus making users of the ontology believe that observers merely host sensors but not actuators.

It seems like we are left with _Host_ and _Platform_ to pick from. Both have their advantages and disadvantages. As we are already making use of the hosts/hostedBy relation, _Host_ seems like a good choice to me. Armin would prefer _Platform_.  We can also put both to the vote as the preferred term for   'the thing that carries/hosts sensors, actuators, and so forth'.

What I hope we can avoid is using multiple terms for a lack of agreement.

Wrt to the example given in: https://github.com/w3c/sdw/issues/309#issuecomment-236809508

> The Observer, e.g. a Policeman makes an Observation based on a cognitive process that a car is driving more than 100 kph, but his colleague standing next to him uses a handheld radar device (device is so much more intuitive, isn't it ;-)) that is a platform that hosts a sensor that implements the sensing of the speed of the object to measure it accurately at 113kph.

The handheld radar device is also an observer as it performs observations (this is a functional definition). Also, if it is a 'platform that hosts', then is takes the thematic role of hosting and thus becomes a host.

IMHO, we should also think back to the target audience. If SOSA-Core is a success, one can easily envision a smartphone's accelerometer generating tens of thousands observations a day but a human (thus, analogue) observation will be orders of magnitude less frequent. Thus, one would expect that billions of observations derived using technical sensors are semantically lifted and published using SOSA-Core on the Web but that observations from human senses, e.g., counting birds, will be less frequent. Thus, I would not recommend having separate Observer and Host/Platform classes (in SOSA-Core) if the sole distinction is human versus artificial.

Long story short, while I prefer _Host_, I am fine with _Platform_ as long as SOSA-Core only has one class to denote 'the thing that carries/hosts sensors, actuators, and so forth'. We (and later users) can define various subclasses to fit their use cases and needs. We can also have a vote on _Platform_ vs _Host_ as labels.

What do you think?

**Comment by KerryLea** [comment]
SSN people,
Again, PLEASE post to the public list unless you intend this conversation to be private and outside the SDW.

Similarly, please use the W3C tracker for issues!  While it is theoretically possible to use github, we agreed to use W3C tracker for this purpose early on, and using both trackers is not sustainable. (Note that the BP  editors and the SDW  made a decision to use github tracker  instead of the W3C tracker  for the BP deliverable alone. However, even in BP work the public list  remains the forum for discussion).

--Kerry

**Comment by maximelefrancois86** [comment]
For the record, SOSA has class sosa:Platform and object property sosa:hosts and sosa:isHostedBy
